pptx/dml/color.py

```diff
@@ -91,6 +91,25 @@ def theme_color(self, mso_theme_color_idx):
             self._color = _SchemeColor(schemeClr)
         self._color.theme_color = mso_theme_color_idx
 
+    @property
+    def preset_color(self):
+        """  Preset Color value of this color.
+        
+        Pone of those defined in the MSO_PRESET_COLOR_INDEX enumeration.  
+        None if no preset color is explicitly defined.  Setting this to
+        a value in MSO_PRESET_COLOR_INDEX causes the color's type to 
+        change to ``MSO_COLOR_TYPE.PRESET``
+        """
+        return self._color.preset_color
+
+    @preset_color.setter
+    def preset_color(self, mso_preset_color_val):
+        # change to theme color format if not already
```

Comment:
Can you do doc string here instead  since it's the first comment on the function? Just to keep things consistent and I think doc string helps vs code pull up a summary of the function.

Reply:
The docstring comes in with the property itself a few lines above, not the setter.  This comment is how it is handled throughout the class.